apps/common/schemas.py

```diff
@@ -58,3 +62,73 @@ class FullNameSchema(BaseOutSchema):
     last_name: str
     first_name: str
     middle_name: str = None
+
+
+class MessageSchema(BaseModel):
+    recipients: List[EmailStr]
+    attachments: List[Union[UploadFile, Dict, str]] = []
+    subject: str = ""
+    body: Optional[Union[str, list]] = None
+    template_body: Optional[Union[list, dict]] = None
+    html: Optional[Union[str, List, Dict]] = None
+    cc: List[EmailStr] = []
+    bcc: List[EmailStr] = []
+    reply_to: List[EmailStr] = []
+    charset: str = "utf-8"
+    subtype: Optional[str] = None
+    multipart_subtype: MultipartSubtypeEnum = MultipartSubtypeEnum.mixed
+    headers: Optional[Dict] = None
+
+    @validator("attachments")
+    def validate_file(cls, v):
+        temp = []
+        mime = MimeTypes()
+
+        for file in v:
+            file_meta = None
+            if isinstance(file, dict):
+                keys = file.keys()
+                if "file" not in keys:
+                    raise WrongFile('missing "file" key')
+                file_meta = dict.copy(file)
+                del file_meta["file"]
+                file = file["file"]
+            if isinstance(file, str):
+                if (
+                    os.path.isfile(file)
+                    and os.access(file, os.R_OK)
+                    and validate_path(file)
+                ):
+                    mime_type = mime.guess_type(file)
+                    f = open(file, mode="rb")
+                    _, file_name = os.path.split(f.name)
+                    u = UploadFile(file_name, f, content_type=mime_type[0])
+                    temp.append((u, file_meta))
+                else:
+                    raise WrongFile(
+                        "incorrect file path for attachment or not readable"
+                    )
+            elif isinstance(file, UploadFile):
+                temp.append((file, file_meta))
+            else:
+                raise WrongFile(
+                    "attachments field type incorrect, must be UploadFile or path"
+                )
+        return temp
+
+    @validator("subtype")
+    def validate_subtype(cls, value, values, config, field):
```

Comment:
u don't use `config, field`, need to clear it

Reply:
This code from official libraries and deleting that arguments may have influence on the process.

Reply:
Tried to delete them and it works. Deleted